cirq_superstaq/daily_integration_test.py

```diff
@@ -17,6 +17,15 @@ def service() -> cirq_superstaq.Service:
     return service
 
 
+def test_ibmq_compile(service: cirq_superstaq.Service) -> None:
+    qubits = cirq.LineQubit.range(2)
+    circuit = cirq.Circuit(cirq_superstaq.AceCRPlusMinus(qubits[0], qubits[1]))
+    out = service.ibmq_compile(circuit, target="ibmq_jakarta_qpu")
+    assert out.duration == 896
```

Comment:
Thoughts on loosening this? If Jakarta calibrations changed so that the calibrated cross-resonance was shorter/longer (happens every now and then when IBM does a system firmware change), the test would fail.

```suggestion
    assert 800 <= out.duration <= 1000  # 896 as of 12/27/2021
```

Reply:
Done.